frontend/src/components/regions/UserDetails/ReviewTrustTable/Presentation.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Chip from '@material-ui/core/Chip';
+
+const columns = [
+    { id: 'title', label: 'Title', minWidth: 170 },
+    { id: 'flavours', label: 'Flavours', minWidth: 100 }
+];
+
+const useStyles = makeStyles({
+
+    flavourChip: {
+        marginRight: '0.5rem',
+    },
+});
+
+const ReviewTrustTable = (props) => {
+
+    const { title, flavours } = props;
+    const classes = useStyles();
+
+    return (
+        <Table stickyHeader aria-label="sticky table">
+            <TableHead>
+                <TableRow>

```

Comment:
We need one row for each ReviewTrust. If I am trusted on 3 projects then I should have 3 rows. If I am trusted on zero projects, I should see zero rows.